Project.toml

```
@@ -1,7 +1,7 @@
 name = "Oscar"
 uuid = "f1435218-dba5-11e9-1e4d-f1a5fab5fc13"
 authors = ["The OSCAR Team <oscar@oscar-system.org>"]
-version = "1.2.0-DEV"
+version = "0.12.0"
```

```suggestion
version = "1.2.0"
```
Can we make that 1.2.0 instead? (including the other places, probably running the script again)

---

d'oh. of course. Argh